diffcp/__init__.py

```diff
@@ -2,6 +2,8 @@
 
 from diffcp.cone_program import solve_and_derivative, \
     solve_and_derivative_batch, \
-    solve_and_derivative_internal, SolverError
+    solve_and_derivative_internal, SolverError, \
+    solve_only_batch, solve_only, \
+    solve_internal
```

Comment:
`SolverError` should still be last

Reply:
My bad - I just pushed that change. Hopefully that's right now.
Also, for learning purposes, is putting `SolverError` last in an `__init__.py` file generally a best practice, or is it a cvxgrp stylistic preference, *i.e.*, in a statement like `from diffcp.cone_program import` ..., is it best to put functions and then classes?

Reply:
It's just a cvxgrp preference. I think Ruff and some other tools have opinions about how imports should be organized.

Reply:
Good to know, thanks!